vms/platformvm/api/static_service.go

```diff
@@ -132,6 +135,15 @@ type PermissionlessValidator struct {
 	Delegators      *[]PrimaryDelegator `json:"delegators,omitempty"`
 }
 
+// GenesisPermissionlessValidator should to be used for genesis validators only.
+type GenesisPermissionlessValidator struct {
+	GenesisValidator
+	RewardOwner        *Owner       `json:"rewardOwner,omitempty"`
+	DelegationFee      json.Float32 `json:"delegationFee"`
+	ExactDelegationFee *json.Uint32 `json:"exactDelegationFee,omitempty"`
+	Staked             []UTXO       `json:"staked,omitempty"`
+}
```

Comment:
this is the main change. `GenesisPermissionlessValidator` comes from `PermissionedValidator`but holds only the fields strictly necessary to genesis. The others are dropped